fuuk/people/models/article.py

```diff
@@ -185,7 +185,7 @@ class Meta:
 
 class ArticleConference(Article):
     class Meta:
-        app_label = 'publications'
+#        app_label = 'publications'
```

Comment:
Because there is no app named publications which is not compatible with Django 1.7. Another option would be to create publications app.